### PR TITLE
Improve broadcaster logging for debugging

### PR DESCRIPTION
**Pull request by joshuacolvin0**

### CODE REVIEW

File: packages/arb-util/broadcastclient/broadcastclient.go [hunk 1]
```diff
@@ -101,15 +101,19 @@ func (bc *BroadcastClient) backgroundReader(messageReceiver chan broadcaster.Bro
...
-
 		res := broadcaster.BroadcastMessage{}
 		err = json.Unmarshal(msg, &res)
 		if err != nil {
 			logger.Error().Err(err).Msg("error unmarshalling message")
 			continue
 		}
 
+		if len(res.Messages) > 0 {
+			logger.Debug().Hex("acc", res.Messages[0].FeedItem.BatchItem.Accumulator.Bytes()).Msg("received batch item")
```
Comment (PlasmaPower) [inline packages/arb-util/broadcastclient/broadcastclient.go:112]: I'm not sure if we ever send more than one message at a time, but we should probably log them or at least how many there are.

**Review by PlasmaPower — CHANGES_REQUESTED** [review]

File: packages/arb-util/broadcastclient/broadcastclient.go (hunk 1 above)
Comment (joshuacolvin0) [reply packages/arb-util/broadcastclient/broadcastclient.go]: sounds good

**Review by PlasmaPower — APPROVED** [review]
LGTM

**Review by hkalodner — APPROVED** [review]
LGTM

**Pull request merged**